keyboards/keebio/bdn9/keymaps/kratsyn/config.h

```diff
@@ -0,0 +1,2 @@
+#define TAP_CODE_DELAY	20
```

Comment:
Headers should have #pragma once guards
```suggestion
#pragma once
#define TAP_CODE_DELAY	20
```